src/view/shell/index.tsx

```diff
@@ -67,6 +68,19 @@ function ShellInner() {
     }
   }, [closeAnyActiveElement])
 
+  React.useEffect(() => {
+    if (currentAccount) {
+      notifications.requestPermissionsAndRegisterToken(currentAccount)
+    }
+  }, [currentAccount])
+
+  React.useEffect(() => {
+    if (currentAccount) {
+      const unsub = notifications.registerTokenChangeHandler(currentAccount)
```

Comment:
This is fine to run even if we don't have perms cause it's just an event emitter.